ci/check/check.go

```diff
@@ -34,22 +34,22 @@ func Check() {
 
 // CheckCopyright checks for copyright headers in files.
 func CheckCopyright() error {
-	return commands.Copyright("./...", "docs")
+	return commands.CopyrightD(".", "docs")
 }
 
 // CheckGoLint reports linting errors in the solution.
 func CheckGoLint() error {
-	return commands.GoLint("./...", "docs")
+	return commands.GoLintD(".", "docs")
 }
 
 // CheckGoVet checks that the source is compliant with go vet.
 func CheckGoVet() error {
-	return commands.GoVet("./...")
+	return commands.GoVet(".")
```

Comment:
Probably should be `commands.GoVetD(".")`

Reply:
There is no `GoVetD`, `go vet` does not play nicely with directories :(